app/controllers/users_controller.rb

```diff
@@ -7,6 +13,11 @@ def show
     redirect_to root_path
   end
 
+  def index
+    @pagy, @users = pagy User.all, items:
+    Settings.page_10
```

Comment:
```suggestion
    @pagy, @users = pagy User.all, items:
      Settings.page_10
```
sai indent

mà ko bjk tới đây em đã dùng đến `scope ` chưa nhỉ, nếu có, thì thường với màn hình list chúng ta ko gọi `.all` mà thường phải sắp xếp theo name hoặc created at ấy thanh niên